db/migrate/20210401210326_create_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :photo_url
+      t.string :encrypted_password
```

Comment:
`encrypted_password` is not necessary at this point as authentication gem will add own required fields to the table.